man/systemd.netdev.xml

```diff
@@ -342,6 +342,24 @@
           </listitem>
         </varlistentry>
         <varlistentry>
+          <term><varname>GroupForwardMask=</varname></term>
+          <listitem>
+            <para>Bitmask to allow forwarding of link local frames with address
+            01-80-C2-00-00-0X on a bridge device. Only values that set bits
+            not matching BR_GROUPFWD_RESTRICTED in net/bridge/br_private.h allowed.
+
+            Default value 0 does not forward any link local frames.
+
+            Restricted bits:
+            0: 01-80-C2-00-00-00 Bridge Group Address used for STP
+            1: 01-80-C2-00-00-01 (MAC Control) 802.3 used for MAC PAUSE
+            2: 01-80-C2-00-00-02 (Link Aggregation) 802.3ad
+
+            Any values not setting these bits can be used. Take special
+            care when forwarding control frames e.g. 802.1X-PAE or LLDP.</para> 
```

Comment:
please use proper docbook markup, and verify that the man pages look OK when building them. For example, paragraphs should be properly encoded in `<para></para>` and so on. The table should be a proper table, and so on.